### ISSUE TITLE
List: redundant parentheses

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/eo-collections#80]
The [list.eo](https://github.com/objectionary/home/blob/master/objects/org/eolang/collections/list.eo) contains lots of redundant parentheses:
```java
[ERROR] net.sf.saxon.style.XSLText@350d31be: Warnings identified: '(acc-index.at 0)' contains redundant parentheses; '(acc-index.at 1)' contains redundant parentheses; '(index.lt (carr.length))' contains redundant parentheses; '(carr.at index)' contains redundant parentheses; '(index.plus 1)' contains redundant parentheses; '(acc-index.at 0)' contains redundant parentheses; '(acc-index.at 1)' contains redundant parentheses; '(index.lt (carr.length))' contains redundant parentheses; '(carr.at index)' contains redundant parentheses; '(index.plus 1)' contains redundant parentheses;
```
It would be good to get rid of them.
You can check warnings by using the latest version of `eo-maven-plugin`.
Comes from: https://github.com/objectionary/eo/issues/1490

**Comment by volodya-lombrozo** [comment]
```sh
[WARNING] Redundant parentheses (acc-index.at 0)
[WARNING] Redundant parentheses (acc-index.at 1)
[WARNING] Redundant parentheses (index.lt (carr.length))
[WARNING] Redundant parentheses (carr.at index)
[WARNING] Redundant parentheses (index.plus 1)
[WARNING] Redundant parentheses (acc-index.at 0)
[WARNING] Redundant parentheses (acc-index.at 1)
[WARNING] Redundant parentheses (index.lt (carr.length))
[WARNING] Redundant parentheses (carr.at index)
[WARNING] Redundant parentheses (index.plus 1)
```

**Comment by Graur** [comment]
@volodya-lombrozo I believe we solved it here #87
